embed/etcd.go

```diff
@@ -123,12 +123,15 @@ func StartEtcd(inCfg *Config) (e *Etcd, err error) {
 		urlsmap types.URLsMap
 		token   string
 	)
-
+	urlsmap, token, err = cfg.PeerURLsMapAndToken("etcd")
 	if !isMemberInitialized(cfg) {
-		urlsmap, token, err = cfg.PeerURLsMapAndToken("etcd")
 		if err != nil {
 			return e, fmt.Errorf("error setting up initial cluster: %v", err)
 		}
+	} else {
+		// if member is already initialized,
```

Comment:
if member is initialized, we should get the peer URLs from on disk configuration, not from cfg (which is from command line flags i assume?)

Reply:
Yes, that's what `PeerURLsMapAndToken` is doing? This is just resetting `token` and `err` as before, since it used to be set only when `!isMemberInitialized(cfg)`